test_contextual_lenses_pooling.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+"""test_contextual_lenses_pooling.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1QqKFn_NllxvCxPaQ24AsKZdISkjG01rr
+
+# Imports
+"""
+
+import functools
+import itertools
+import os
+import time
+
+import flax
+from flax import jax_utils
+from flax import nn
+from flax import optim
+
+import jax
+from jax import random
+from jax import lax
+import jax.nn
+import jax.numpy as jnp
+
+import numpy as np
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+import scipy.stats
+
+import sklearn.linear_model
+import sklearn.metrics
+
+import seaborn as sns
+
+from absl.testing import parameterized
+
+"""# Contextual lenses"""
+
+def max_pool(x, padding_mask, rep_size=None):
```

Comment:
you shouldn't be defining max_pool, etc. here. you should be importing from contextual_lenses.py